apps/researcher/src/app/[locale]/(objects)/heritage-object-card.tsx

```diff
@@ -3,44 +3,60 @@ import {useTranslations} from 'next-intl';
 import {HeritageObject} from '@/lib/objects';
 import Image from 'next/image';
 import {encodeRouteSegment} from '@/lib/clerk-route-segment-transformer';
+import classNames from 'classnames';
 
 interface Props {
   heritageObject: HeritageObject;
 }
 
 export default function HeritageObjectCard({heritageObject}: Props) {
   const t = useTranslations('HeritageObjectCard');
+  const imageUrl =
+    heritageObject.images && heritageObject.images.length > 0
+      ? heritageObject.images[0].contentUrl
+      : null;
```

Comment:
Out of curiosity: when do you use `null` and when `undefined`?

Thinking out load: if `imageUrl` is `null` it suggests that it has a (valid, empty) value. If `imageUrl` is `undefined` it suggests that it has no value.

If I read line 39 correctly, the check for `imageUrl` is not about a specific value but about existence: does an image URL exist?